app/controllers/state_file/questions/nc_veteran_status_controller.rb

```diff
@@ -0,0 +1,11 @@
+module StateFile
+  module Questions
+    class NcVeteranStatusController < QuestionsController
+      include ReturnToReviewConcern
+
+      def update
+        update_for_device_id_collection(current_intake&.initial_efile_device_info)
```

Comment:
[DUST] remind me how we decide where to put this? it seems like the pattern is that it goes after the data transfer but why not `NameDobController`? (i see that for AZ it's `AzPriorLastNamesController` which is also not the first page after data transfer)

Reply:
oh we probably dont need this in this controller since it looks like there's another page (nc county) before this